src/components/Carousel/index.js

```diff
@@ -149,6 +152,7 @@ export default class Carousel extends PureComponent {
           slidesToScroll={scrollCount}
           slidesToShow={showCount}
           infinite={loop}
+          dots={dots}
```

Comment:
Should this be `{...dots}` so we can pass in custom options like `dotsClass` in addition / make our own adjustments to dots?

Reply:
Actually, the changes to this file to add passing along of dots isn't necessary.  `{...restProps}` below passes all props.  The only reason we pass the props above is because the names are changed.